build-tools/packages/build-cli/api-report/build-cli.api.md

```diff
@@ -32,12 +32,18 @@ export interface FlubConfig {
     bump?: BumpConfig;
     policy?: PolicyConfig;
     releaseNotes?: ReleaseNotesConfig;
+    releaseReport?: LegacyCompatInterval;
     version?: 1;
 }
 
 // @public
 export const knownReleaseGroups: readonly ["build-tools", "client", "server", "gitrest", "historian"];
 
+// @public
+export interface LegacyCompatInterval {
```

Comment:
This looks stale? CI doesn't catch these out of date?

Reply:
On running build locally, there are no api-report changes.